### PR TITLE
Change contributing workflow

### PR DESCRIPTION
**Pull request by sappenin**
This PR re-organizes the Contributing guidelines to reflect the process agreed upon by @Silkjaer, @intelliot , @sappenin, @amarantha-k and a few others in April of 2023 (but which was never update in this repo).

Given that it has unfortunately been some time since this discussion took place, if you would like to propose any changes to this PR, please feel free.

### CODE REVIEW

**Comment by sappenin** [comment]
Bumping this once more - I realize these changes are quite old, so am open to feedback, suggestions, debate, etc.

1. While there are a lot of changes in this PR, most of these changes are simply moving text around for clarity and improved organization. 
2. The existing `README.md` and `CONTRIBUTING.md` files in `master` have similar -- though not always the same -- content, which may cause confusion or ambiguity. This PR prefers to move any content related to contributions into `CONTRIBUTING.md`, for clarity. 
3. In the intervening months since the changes in this PR were first proposed, some different changes were merged into the `README`. E.g., see [here](https://github.com/XRPLF/XRPL-Standards/commit/0f6261182b4c6c744da1de81712dccb1f0ac3ffc), [here](https://github.com/XRPLF/XRPL-Standards/commit/af77fb786a5ad7f0727f37900393f5bba2702d2f), and [here](https://github.com/XRPLF/XRPL-Standards/commit/a3cbc646f9d49ffcb062bc49751331d8b005a05f). I have incorporated these into `CONTRIBUTING.md` in this PR. If I missed anything, please suggest a change. @WietseWind @RichardAH 
5. I have only medium conviction around these changes, so while I think they improve the overall process, I won't be offended if anyone wants to suggest changes to what is proposed here.

@Silkjaer @intelliot

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,78 +1,66 @@
...
+#### Specification Number
+Use the next number that has not yet been used. If a conflict occurs, it will be fixed by a maintainer or editor. Maintainers or editors also reserve the right to remove or re-number proposals as necessary. The number is important, as it will be used to reference features and ideas throughout the community.
 
+### 2. Closing Discussion
 
-## Process
+When a discussion has produced a well-refined standard, authors should post a comment to the discussion noting that the discussion will be closed in a few days. This allows time (for those engaged with the discussion) to submit final commentary. 
 
-The XRPL-Standards process attempts to be easy to use, but also rigorous enough that there are permalinks to revisions of documents for reference purposes.
+Once this waiting period has elapsed, the standard's author may close the discussion from further comment, and then move the discussion to a SPECIFICATION PULL REQUEST to add the standard into the repository as a markdown file (see below for specification formats).
```
Comment (mvadari) [inline CONTRIBUTING.md:27]:
```suggestion
Once this waiting period has elapsed, the standard's author may close the discussion from further comment, and then move the discussion to a [**specification pull request**](#3-specification-pull-requests) to add the standard into the repository as a markdown file (see below for specification formats).
```
(I might have gotten the link format wrong to link to the below section)

**Review by mvadari — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (sappenin) [reply CONTRIBUTING.md]: Great suggestion - added via 248828e94151197f46cdc619488b9ae19f6a17db

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,78 +1,66 @@
...
+#### Specification Number
+Use the next number that has not yet been used. If a conflict occurs, it will be fixed by a maintainer or editor. Maintainers or editors also reserve the right to remove or re-number proposals as necessary. The number is important, as it will be used to reference features and ideas throughout the community.
 
+### 2. Closing Discussion
 
-## Process
+When a discussion has produced a well-refined standard, authors should post a comment to the discussion noting that the discussion will be closed in a few days. This allows time (for those engaged with the discussion) to submit final commentary. 
 
-The XRPL-Standards process attempts to be easy to use, but also rigorous enough that there are permalinks to revisions of documents for reference purposes.
+Once this waiting period has elapsed, the standard's author may close the discussion from further comment, and then move the discussion to a [**specification pull request**](#3.-specification-pull-requests) to add the standard into the repository as a markdown file (see below for specification formats).
```
Comment (ximinez) [inline CONTRIBUTING.md:27]:
```suggestion
Once this waiting period has elapsed, the standard's author may close the discussion from further comment, and then move the discussion to a [**specification pull request**](#3-specification-pull-requests) to add the standard into the repository as a markdown file (see below for specification formats).
```
(Need to remove the "." from the anchor.)

Note that you can get the anchor by clicking the 3-dots menu, selecting "View file", navigating to the section you want the anchor for, and hovering your mouse just to the left of it. A link icon will appear, which you can right-click on to copy the URL. (For this section right now, it's https://github.com/XRPLF/XRPL-Standards/blob/248828e94151197f46cdc619488b9ae19f6a17db/CONTRIBUTING.md#3-specification-pull-requests) Then the URL only needs the anchor portion.

Comment (sappenin) [reply CONTRIBUTING.md:27]: Oh, thanks, and nice catch -- that's super helpful!

**Review by mvadari — APPROVED** [review]

**Review by amarantha-k — APPROVED** [review]
Thanks for the updates, @sappenin! Other than a minor suggestion, looks good.

**Review by intelliot — APPROVED** [review]

**Pull request merged**